webapp/components/admin_console/developer_settings.jsx

```diff
@@ -48,13 +48,13 @@ export default class DeveloperSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.testingTitle'
-                            defaultMessage='Enable Testing: '
+                            defaultMessage='Enable Testing Commands: '
                         />
                     }
                     helpText={
                         <FormattedMessage
                             id='admin.service.testingDescription'
-                            defaultMessage='(Developer Option) When true, /loadtest slash command is enabled to load test accounts and test data. Changing this will require a server restart before taking effect.'
+                            defaultMessage='When true, /loadtest slash command is enabled to load test accounts and test data. This setting also hides any features for disabling text formatting. Changing this will require a server restart before taking effect.'
```

Comment:
Not sure what this means "This setting also hides any features for disabling text formatting."